lib/database/game.php

```diff
@@ -865,9 +865,16 @@ function getTotalUniquePlayers($gameID, $requestedBy)
     return $data['UniquePlayers'];
 }
 
-function getGameTopAchievers($gameID, $offset, $count, $requestedBy)
+function getGameTopAchievers($gameID, $offset, $count, $requestedBy, $totalPossible, $order)
```

Comment:
This function is used in `public/API/API_GetGameRankAndScore.php`, too (found no other additional usage). Please change its usage there, too, with all parameters - ideally to match its current behaviour.

Reply:
Fixed. I removed the $totalPossible parameter and built it into the SQL query. This way the API can be used to get both sets of data being returned from that function by specifying the type in the call. High scores will be returned by default if a no type is specified in the call so it will not break anything currently using the API_GetGameRankAndScore.php.